api-server/src/server/utils/middleware.js

```diff
@@ -77,6 +77,16 @@ export function ifUserRedirectTo(status) {
   };
 }
 
+export function ifNotMobileRedirect() {
+  return (req, res, next) => {
+    const isMobile = /(iPhone|iPad|Android)/.test(req.headers['user-agent']);
+    if (!isMobile) {
+      res.json({ error: 'not from mobile' });
+    } else {
+      next();
+    }
```

Comment:
```suggestion
    //
    // Todo: Use the below check once we have done more research on usage
    //
    // const isMobile = /(iPhone|iPad|Android)/.test(req.headers['user-agent']);
    // if (!isMobile) {
    //  res.json({ error: 'not from mobile' });
    // } else {
    //  next();
    // }
    next();
```

I am having second thoughts about using this because, on the surface, it looks very brittle. There are several other devices that we would need to account for. 

Also, regex is a bad idea in principle as the device list grows.

Let's shelf this for now and revisit it in the future.

Reply:
Cool that sounds good to me 👍